examples/QmlBarcodeReader/qml/ScannerPage.qml

```diff
@@ -88,4 +88,26 @@ ApplicationWindow {
       }
     }
   }
+
+  GeneratorPage {
+      id: generator
+      visible: false
+      onVisibleChanged: generateCode.visible = !generateCode.visible ? true : false
```

Comment:
1. Wrap with braces
2. generateCode.visible has boolean type already, so you can just write: generateCode.visble = !generateCode.visible